examples/game/chess.rs

```diff
@@ -0,0 +1,431 @@
+use bevy::prelude::*;
+
+fn main() {
+    App::new()
+        .insert_resource(Msaa { samples: 4})
+        .insert_resource(WindowDescriptor {
+            title: "Chess!".to_string(),
+            width: 1600.0,
+            height: 1600.0,
```

Comment:
I find 1600x1600 too big. Preferable we would stay at the same window size as the other examples. They use the default 1280x720.